CHANGELOG.md

```diff
@@ -21,6 +21,10 @@ Current
 
 ### Changed:
 
+- [Include ETags on responses](https://github.com/yahoo/fili/pull/417)
+    * Let clients know that weather a response is cached(new) or fetched from cache offers a lot of ability for
```

Comment:
know whether*

Reply:
How about a description that includes the context that this only applies to FullResponses.
"'Modify FullResponse JSON Objects to contain a flag showing whether a response is new or fetched from cache."